notification-portlet-webapp/src/main/webapp/scripts/modal.js

```diff
@@ -0,0 +1,128 @@
+Vue.component("modal", {
+  template: "#modal-template",
+  props: {
+    id: String
+  }
+})
```

Comment:
Prettier is nice :+1: 
I've been hoping to get it rolled out to more uPortal projects. ([ref 1](https://github.com/Jasig/uPortal/blob/9c4774f7449c0de4da830791f6698015b9ab7e41/package.json#L21), [ref 2](https://github.com/ChristianMurphy/widget-creator/blob/ce57497cb5a117810a8afff5679bbce06ef97894/package.json#L14)) :smile: 

Let's leave it as is for now, if there is a request to get the style changed, `prettier-eslint` with `eslint-config-google` can automate the transition. :robot:

Reply:
Since @ChristianMurphy *did* conjure me here: "Adopt Google Style."

Universally adopting Google style conventions is a way to avoid arguing about which style conventions to adopt, across a variety of languages and technologies, and thereby across a breadth of uPortal Ecosystem projects. So yes, I have a strong opinion that the correct answer is "Adopt Google Style.". Fits on an index card I keep in my wallet as a handy reminder.

That said, MyUW hasn't adopted `NotificationPortlet` so I don't feel I have anything like enough investment here to insist upon this. 🤷‍♂️

Reply:
Thanks @apetro for explaining the benefit of Google Style over others. While a name brand brings along some value in that developers may be familiar with it from other projects, the fact that it applies to a variety of languages and technologies is an excellent reason to adopt it.

Reply:
For now I think we should leave the code style as is, when I applied google code style over in #61 it broke the minification, we would need an ES6+ compatible minifier to fully apply the google code style.
That can be addressed in the future.